src/setupTests.ts

```diff
@@ -3,3 +3,6 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import "@testing-library/jest-dom";
+
+const noop = () => {};
+Object.defineProperty(window, "scrollTo", { value: noop, writable: true });
```

Comment:
Why do we need this?

Reply:
When I using yarn test to test my unit test of richeditor, the testcases passed 100% but some errors happened.
![image](https://user-images.githubusercontent.com/58205909/121802474-93015280-cc66-11eb-8f91-b7ebcc63c77c.png)
When I read documents, I realiezed I need to mock window.scrollTo in each test case need to use ref.
![image](https://user-images.githubusercontent.com/58205909/121803135-4586e480-cc6a-11eb-8c56-e6df52a2a96a.png)
So i set here to effect all test case